xclim/indicators/atmos/_precip.py

```diff
@@ -247,6 +250,31 @@ class HrPrecip(Hourly):
     parameters={"tas": {"kind": InputKind.VARIABLE}, "phase": "solid"},
 )
 
+standardized_precipitation_index = PrecipWithIndexing(
+    title="Standardized Precipitation Index (SPI)",
+    identifier="spi",
+    units="",
+    standard_name="spi",
+    long_name="Standardized Precipitation Index (SPI)",
+    description="Precipitations over rolling window {window}-X window, normalized such that SPI averages to 0. The window unit `X` is the minimal time period defined by resampling frequency {freq}",
```

Comment:
```suggestion
    description="Precipitations over a rolling window of {window} {freq:noun}, normalized such that SPI averages to 0.",
```
Pareil. Ici il faudra ajouter `"D": ["daily", "days"]` dans `xclim.formatting.default_formatter`.

Reply:
The SPI index is guaranteed to be centered on 0 by definition: The CDF (from whatever distribution is chosen) is inverted back to a normal distribution. So I don't think we need to ask for a constraint on the calibration data. Or did I misunderstand your point?

Reply:
ok. I might be confused about what we're discussing but looking at SPEI data calculated for the climatedata.ca project. I see that output spei values are centred more or less on 0 for 'calibration' period (historical grey data in this case) 
https://climatedata.ca/explore/variable/?coords=64.69793160441417,-103.88397216796876,8&delta=&geo-select=&var=spei_3m&var-group=precipitation&mora=jul&rcp=rcp85&decade=2070s&sector=

![image](https://user-images.githubusercontent.com/22454970/176520185-e7d0f3c1-5a91-46ca-b027-8f9d8ccc4b3c.png)

I still think having the calibration period in the metadata somewhere would be an important piece of info to retain
(From what I see in the signature the `history` attr will only tell us that there was an `xr.DataArray` used)

Reply:
I checked and the french translations are working well but I think you forgot to change the english version as well.